tests/integration/optimizers/test_optimizer.py

```diff
@@ -0,0 +1,12 @@
+from .optimize import main
+
+
+def test_optimizer():
+    """This will run complete optimisation.
+    Todo: fix number of index and query doc so that test doesnt run long
+    Todo: assert to check eval key and value for values generated in config/best_config.yml
+    Todo: after optimisation is complete, assert to check trial pods and flows do not have any evironment variable.
+          This can be checked by some of our current yaml related functions in flow runner
+          and changing its functionality to assert values are not starting with $.
+    """
+    main()
```

Comment:
put the main directly here

Reply:
Yes that's the plan once everything is ready